tests/validation/cattlevalidationtest/core/test_ldap.py

```diff
@@ -0,0 +1,181 @@
+
+    def __call__(self, r):
+        # modify and return the request
+        r.headers['Authorization'] = 'Bearer ' + self.jwt
+        if self.prj_id is not None:
+            r.headers['X-API-Project-Id'] = self.prj_id
+        return r
+
+
+def load_config():
+    config = {
+        "accessMode": "unrestricted",
+        'domain': os.environ.get('API_AUTH_LDAP_DOMAIN', "dc=rancher,dc=io"),
+        'groupNameField': os.environ.get('API_AUTH_LDAP_GROUP_NAME_FIELD',
+                                         'name'),
+        'groupObjectClass': os.environ.get('API_AUTH_LDAP_GROUP_OBJECT_CLASS',
+                                           'group'),
+        'groupSearchField': os.environ.get('API_AUTH_LDAP_GROUP_SEARCH_FIELD',
+                                           'sAMAccountName'),
+        'loginDomain': os.environ.get('API_AUTH_LDAP_LOGIN_NAME', 'rancher'),
+        'port': os.environ.get('API_AUTH_LDAP_PORT', 389),
+        'server': os.environ.get('API_AUTH_LDAP_SERVER', 'ad.rancher.io'),
+        'serviceAccountPassword': os.environ.get('API_AUTH_LDAP_'
+                                                 'SERVICE_ACCOUNT_PASSWORD',
+                                                 'Password1'),
+        'serviceAccountUsername': os.environ.get('API_AUTH_LDAP_'
+                                                 'SERVICE_ACCOUNT_USERNAME',
+                                                 'cattle'),
+        'tls': False,
+        'userDisabledBitMask': os.environ.get('API_AUTH_LDAP_'
+                                              'USER_DISABLED_BIT_MASK',
+                                              '2'),
+        'userEnabledAttribute': os.environ.get('API_AUTH_LDAP_'
+                                               'USER_ENABLED_ATTRIBUTE',
+                                               'userAccountControl'),
+        'userLoginField': os.environ.get('API_AUTH_LDAP_USER_LOGIN_FIELD',
+                                         'sAMAccountName'),
+        'userNameField': os.environ.get('API_AUTH_LDAP_'
+                                        'USER_NAME_FIELD', 'name'),
+        'userObjectClass': os.environ.get('API_AUTH_LDAP_USER_OBJECT_CLASS',
+                                          'person'),
+        'userSearchField': os.environ.get('API_AUTH_LDAP_USER_SEARCH_FIELD',
+                                          'name')
+    }
+    return config
+
+
+@pytest.fixture(scope='module', autouse=True)
+def ldapConfig(admin_client):
+    config = load_config()
+    admin_client.create_ldapconfig(config)
+    serviceAccountDN = os.getenv('API_AUTH_LDAP_SERVICE_ACCOUNT_DN',
+                                 "cn=Cattle,"
+                                 "ou=Rancher Labs,dc=rancher,dc=io")
+    x = admin_client.by_id('identity', 'ldap_user:' + serviceAccountDN)
+    assert x.login == config['serviceAccountUsername']
+
+
+@if_ldap
+def test_ldap_search_get_user(admin_client):
+    searchUserlogin = os.getenv('LDAP_USER1', 'sangeetha')
```

Comment:
can you not use any team members actual names in the tests? Bummer if that means rebuilding the AMI, but I think it is inappropriate to have our names in here.